arcgis-ios-sdk-samples/Route & Directions/Find closest facility to an incident (interactive)/FindClosestFacilityInteractiveViewController.swift

```diff
@@ -0,0 +1,122 @@
+//
+// Copyright © 2020 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import UIKit
+import ArcGIS
+
+class FindClosestFacilityInteractiveViewController: UIViewController {
+    @IBOutlet var mapView: AGSMapView! {
+        didSet {
+            // Initialize the map.
+            mapView.map = AGSMap(basemapType: .streets, latitude: 32.727, longitude: -117.1750, levelOfDetail: 12)
+            mapView.touchDelegate = self
+            
+            // Create symbols and graphics to add to the graphic overlays.
+            mapView.graphicsOverlays.add(makeFacilitiesOverlay())
+            mapView.graphicsOverlays.add(incidentGraphicsOverlay)
+        }
+    }
+    
+    private let facilitySymbolURL = URL(string: "https://static.arcgis.com/images/Symbols/SafetyHealth/Hospital.png")!
```

Comment:
I didn't think about it at the time, but this is only being used inside `makeFacilitiesOverlay()`, so you could inline it there if you wanted to. That would probably be a better place for it.

Reply:
Could you clarify what you mean by "inline?" Do you mean making it a closure?

Reply:
Sure. I just meant that you could move it into the `makeFacilitiesOverlay()` function:

```swift
private func makeFacilitiesOverlay() -> AGSGraphicsOverlay {
    let facilitySymbolURL = URL(string: "https://static.arcgis.com/images/Symbols/SafetyHealth/Hospital.png")!
    let facilitySymbol = AGSPictureMarkerSymbol(url: facilitySymbolURL)
```

Reply:
I see now; I misunderstood your original comment!